lib/ddtrace/contrib/rails/active_record.rb

```diff
@@ -52,6 +55,25 @@ def self.sql(_name, start, finish, _id, payload)
         rescue StandardError => e
           Datadog::Tracer.log.error(e.message)
         end
+
+        def self.instantiation(_name, start, finish, _id, payload)
+          tracer = Datadog.configuration[:rails][:tracer]
+          rails_service = Datadog.configuration[:rails][:service_name]
+
+          span = tracer.trace(
+            'active_record.instantiation',
+            resource: payload.fetch(:class_name),
+            service: rails_service,
```

Comment:
Have to change this... I think it should belong to the parent service (if available), to be consistent with the behavior in standalone ActiveRecord. It will fallback to this is the parent name isn't available.